site2/website/blog/2022-02-24-apache-pulsar-community-welcomes-500th-contributor

```diff
@@ -0,0 +1,60 @@
+
+## Product Updates and Ecosystem Development
+
+The Apache Pulsar community released versions [2.8.0](https://pulsar.apache.org/blog/2021/09/23/Apache-Pulsar-2-8-1/) in June 2021, and [2.9.0](https://pulsar.apache.org/release-notes/#281-mdash-2021-09-10-a-id281a) in December 2021, which contain a number of new benefits, including: 
+
+1. **Enhanced support for protocol handlers** enables teams to integrate existing applications built on other technologies with Pulsar without code changes. Popular protocol handlers developed by the community including:
+   * [Kafka-on-Pulsar (KoP)](https://hub.streamnative.io/protocol-handlers/kop/2.8.1.2)
+   * [RabbitMQ (AoP)](https://hub.streamnative.io/protocol-handlers/aop/0.1.0)
+
+2. Developers can now utilize **Pulsar integrations** with multiple processing engines and databases, including [Flink](https://flink.apache.org/2021/01/07/pulsar-flink-connector-270.html), [Spark](https://github.com/streamnative/pulsar-spark), [DeltaLake](https://delta.io/uncategorized/pulsar-in-the-lakehouse-apache-pulsar-with-apache-spark-and-delta-lake/), [Druid](https://imply.io/blog/community-spotlight-apache-pulsar-and-apache-druid-get-close/)), [Trino](https://www.youtube.com/watch?v=Z77DqONXyi4), and [Pinot](https://github.com/apache/pinot/pull/7026). 
+
+3. **Cluster Failover** [[PIP-121]](https://github.com/apache/pulsar/pull/13316) provides an automatic cluster level recovery mechanism that automatically redirects all Pulsar clients to a standby cluster in the event of catastrophic cluster failure of the active Pulsar cluster. This helps minimize the downtime experienced by the client applications by allowing them to seamlessly transition from one Pulsar to the next without user intervention.
+
+4. **Improved multi-tenancy support with tenant and namespace rate limiting.** [[PIP-82]](https://github.com/apache/pulsar/pull/11918) This feature allows users to configure a namespace or tenant wide limit for producers and consumers, and have that enforced irrespective of the number of topics in the namespace, allowing control of the max rate across a given namespace. 
+
+5. **Simplified microservices development** with Pulsar’s high-level APIs is now possible. [Exclusive producers](https://www.youtube.com/watch?v=Chw8VBS9WfI) enable developers to build coordination and consensus mechanisms.
+
+6. The new [Transactions feature](https://streamnative.io/blog/release/2021-06-14-exactly-once-semantics-with-transactions-in-pulsar/) empowers organizations to use Pulsar as a system-of-record that enables event-based architectures, similar to those popular in finance and real-time systems. With this update, Pulsar's message delivery/processing semantic evolved from message deduplication on a single topic, to atomic produce and acknowledgment over multiple topics. 
```

Comment:
Again you are referring to a SN blog post which does not link back in any way to the Apache Pulsar project.